app/services/region_backfill.rb

```diff
@@ -115,4 +115,44 @@ def count_invalid(region_type)
     invalid_counts[region_type] ||= 0
     invalid_counts[region_type] += 1
   end
+
+  # Wraps a Region object and allows us to easily avoid persistence
+  # methods when we are exercising Regions in a backfill dry_run scenario
+  class DryRunRegion < SimpleDelegator
```

Comment:
Removed the `DryRunRegion` model and inlined it into `RegionBackfill` script. I've kept the script around just in case we need it in some off chance something is broke, but reduced the amount of footprint in the codebase.